core/src/mindustry/world/blocks/units/CommandCenter.java

```diff
@@ -63,7 +63,11 @@ public void draw(){
 
             Draw.color(bottomColor);
             Draw.rect(commandRegions[team.data().command.ordinal()].getRegion(), tile.drawx(), tile.drawy() - 1, size, size);
-            Draw.color(topColor);
+            if(topColor == null){
+                Draw.color(team.color);
+            }else{
+                Draw.color(topColor);
+            }
```

Comment:
not sure which one anuke prefers, but this one makes the draw function a little more compact:
```suggestion
            Draw.color(topColor == null ? team.color : topColor);
```

Reply:
Oh, thx